README.md

```diff
@@ -16,15 +22,28 @@
 6. `npm run dev` or `yarn run dev` to run it in development mode.
 7. Go to `http://localhost:{{PORT}}/`   =>{{PORT}} : is the port number that you precise in your `.env` file
 
-### C. Tools used
+## Description 
+
+### Features
 1. [Webpack 4](https://webpack.js.org/) for bundling
 2. [Express](https://expressjs.com/) for serving the frontend
 3. [GitHub and Git](https://github.com/) for Version control
 4. [TravisCI](https://travis-ci.org/) for continuous integration
 5. [JestJS](https://jestjs.io/docs/en/webpack) for testing
 6. [Enzyme](https://enzymejs.github.io/enzyme/docs/installation/) for DOM testing
 
-### D. Dependencies
+## Dependencies
 1. [node-sass](https://www.npmjs.com/package/node-sass)
 2. [sass](https://www.npmjs.com/package/sass)
 3. [sass-loader](https://www.npmjs.com/package/sass-loader)
+
+## Test
+> *   run test using `npm test`
+> *   run test in watch mode using `npm run test:watch`.
+> *   run test coverage using `npm run test:coverage`.
+
+[![Maintainability](https://api.codeclimate.com/v1/badges/a99a88d28ad37a79dbf6/maintainability)](https://codeclimate.com/github/codeclimate/codeclimate/maintainability)
+
+[![Test Coverage](https://api.codeclimate.com/v1/badges/a99a88d28ad37a79dbf6/test_coverage)](https://codeclimate.com/github/codeclimate/codeclimate/test_coverage)
+
+[![Build Status](https://travis-ci.org/Stackup-Rwanda/stackup2-barefoot-frontend.svg?branch=develop)](https://travis-ci.org/Stackup-Rwanda/stackup2-barefoot-frontend)
```

Comment:
Don't place badges at this place, badges are better placed below the project's title.